componentLibraries/defaultLibrary/Signal/Control/SignalPID2.hpp

```diff
@@ -69,8 +69,10 @@ class SignalPID2 : public ComponentSignal
         addConstant("Ti", "Integral time", "s", 1, mTi);
         addConstant("Tt", "Anti-windup tracking constant", "s", 1, mTt);
         addConstant("Td", "Derivative time", "s", 0, mTd);
-        addConstant("Umin", "", "", -1e100, mUmin);
-        addConstant("Umax", "", "", 1e100, mUmax);
+        addConstant("Umin", "Minimum output for anit-windup", "", -1e100, mUmin);
```

Comment:
Maybe "Lower anti-windup limit" and "Upper anti-windup limit", as the anti-windup think is an internal thing. Someone not accustom to anti-windup may misinterpret the work output in this case.

Reply:
I referred to the "output" to distinguish it from having anything to do with maximum/minimum value of the integrator part. I.e. the parameter does not limit the size of the integrator part, it just limits the (output) conditions under which it's allowed to "work". Again, @peterNordin and @robbr48 just let me know what you prefer.

Reply:
I still think that the description is confusing, but you are correct, only proper documentation can address this and I have no better idea myself. I might adjust the description slightly when I get around to writing that documentation.